brave/browser-laptop
# Add English (U.K.) language

Auditors: @kjozwiak, @bsclifton 

Test Plan:

  * Start browser, navigate to about:preferences
  * Change language setting to English (U.K.), restart browser
  * Ensure language has changed

Implements: #10623

Submitter Checklist:
